### ISSUE TITLE
Attributes without quotations are not reported as errors

### ISSUE THREAD

**Issue by MeirionHughes** [MeirionHughes/aurelia-template-lint#127]
```gherkin
Given <require from=somepath></require>
When parsed by linter
Then an error should occur
```
Currently fails due to [parse5 SAXParser successfully parsing](https://github.com/inikulin/parse5/issues/157) the `somepath` and returning it as a string.
